### PR TITLE
feat(access):  mark sealed volume when write failed. for blobstore-v1.3.0 feature branch

### PR DESCRIPTION
**Pull request by mawei029**
**What this PR does / why we need it**:
1. mark sealed volume, when access stream wirte failed

**Which issue this PR fixes**:

fixes #

**Special notes for your reviewer**:
I will merge multiple commits when the code view passes

**Release note**:

```release-note
```

### CODE REVIEW

File: blobstore/access/stream_put.go [hunk 1]
```diff
@@ -382,18 +387,35 @@ func (h *Handler) writeToBlobnodes(ctx context.Context,
 			st := <-statusCh
 			received[st.index] = st
 		}
-
 		if _, ok := <-writeDone; !ok {
 			return
 		}
 
+		_, ok := h.failVids.Load(vid)
```
Comment (sejust) [inline blobstore/access/stream_put.go:394]: LoadOrStore

**Review by sejust — CHANGES_REQUESTED** [review]

File: blobstore/access/stream_put.go [hunk 2]
```diff
@@ -436,5 +458,38 @@ func (h *Handler) writeToBlobnodes(ctx context.Context,
 
 	close(writeDone)
 	err = fmt.Errorf("quorum write failed (%d < %d) of %s", writtenNum, putQuorum, blob.String())
+	// need mark sealed: less than quorum, az fail
+	releaseType = releaseVolumeSealed
 	return
 }
+
+type failVuidMap struct {
```
Comment (JasonHu520) [inline blobstore/access/stream_put.go:466]: 可以直接用sync.Map，计数就用原子操作就可以？

Comment (mawei029) [reply blobstore/access/stream_put.go:462]: 这里的结构体是在395行的sync.Map里value用的，sync.Map不能嵌套

**Review by sejust — APPROVED** [review]

File: blobstore/access/stream_put.go (hunk 1 above)
Comment (mawei029) [reply blobstore/access/stream_put.go]: ok

File: blobstore/access/stream_put.go (hunk 2 above)
Comment (mawei029) [reply blobstore/access/stream_put.go:462]: ok

**Review by JasonHu520 — APPROVED** [review]

**Pull request merged**